templates/history-downloads.php

```diff
@@ -71,7 +71,7 @@
 										endforeach;
 
 									else :
-										_e( 'No downloadable files found.', 'easy-digital-downloads' );
+										apply_filters('edd_receipt_no_files_found_text', __( 'No downloadable files found.', 'easy-digital-downloads' ), $download['id']);
```

Comment:
@Jo-IE It looks like this won't get output to the screen because the `_e` function has been removed. You'll need to replace it with an `echo` command so that it gets output to the screen once the filter has been applied.

Reply:
ok, I see. I will get on it right away. Thank you @mintplugins !